example/multiphase/pseudo_potential/pp.xml

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0"?>
+<CLBConfig output="output/">
+        <Geometry nx="128" ny="128">
+        <MRT><Box/></MRT>
+                <Wall mask="ALL" name="border" >
+                        <Box nx="1"/>
+                        <Box dx="-1"/>
+                        <Box ny="1"/>
+                        <Box dy="-1"/>
+                </Wall>
+                <None name="blobb">
+                        <!-- <Box dx="50" nx="28" dy="50" ny="28"/> -->
+                        <Sphere ny="48" nx="48" dx="40" dy="0" /> 
+                </None>
+        </Geometry>
+        <Model>
+                <Params Density="0.056"/>
+                <Params Density-blobb="2.659"/>
+                <Params G_ff="-6.0"/>
+                <Params G_sf="-3.7"/>
+                <Params nu="0.166"/>
+                <!-- <Params GravitationY="-1e-5"/> -->
+        </Model>
+        <!-- <Solve Iterations="100" output="output/"> <VTK Iterations="1"/> </Solve>  -->
+        <Solve Iterations="1000" output="output/"> <VTK Iterations="10"/> </Solve> 
+        <Failcheck Iterations="1000" />
```

Comment:
Add box to failcheck - no need to evaluate whole domain, especially walls

Reply:
So what exactly the empty <Box/> element does?

Reply:
I think @mdzik is refering to this: https://github.com/CFD-GO/TCLB/blob/develop/src/Handlers/cbFailcheck.cpp.Rt#L18
You can specify a region to check like this: `<Failcheck ... dx="10" nx="10"/>`

On the other hand, I would not consider it crusial.